docs/pages/versions/unversioned/sdk/auth-session.md

```diff
@@ -547,7 +547,7 @@ A hook used for opinionated Google authentication that works across platforms.
 
 #### Returns
 
-- **request (_GoogleAuthRequest | null_)** -- An instance of [`GoogleAuthRequest`](#googleauthrequest) that can be used to prompt the user for authorization. This will be `null` until the auth request has finished loading.
+- **request (_GoogleAuthRequest | null_)** -- An instance of [`GoogleAuthRequest`](#useauthrequest) that can be used to prompt the user for authorization. This will be `null` until the auth request has finished loading.
```

Comment:
@EvanBacon I don't think `GoogleAuthRequest` is defined in the docs, only `GoogleAuthRequestConfig`. Do you know if this should be added or where this should link to?